packages/web-components/src/components/va-minimal-header/va-minimal-header.tsx

```diff
@@ -0,0 +1,36 @@
+import { Component, Host, h, Prop} from '@stencil/core';
+
+@Component({
+  tag: 'va-minimal-header',
+  styleUrl: 'va-minimal-header.scss',
+  shadow: true,
+  assetsDirs: ['./assets'],
+})
+
+export class VaMinimalHeader {
+  @Prop() header?: string;
+  @Prop() subheader?: string;
+
+  render() {
+    const {header, subheader} = this;
+    
+    return (
+      <Host>
+        <va-official-gov-banner/>
+        <va-crisis-line-modal/>
```

Comment:
I wonder if it would be useful to have `va-crisis-line-modal` as its own standalone component (instead of embeded)? 🤔

Reply:
After I commented there I remember that we do have the crisis modal already a variation of va-modal. Is it possible to just use this?

https://minimal-header-dst2030--60f9b557105290003b387cd5.chromatic.com/?path=/docs/components-va-modal--crisis-line-modal

Reply:
We have [a ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2016) to update that modal to match the crisis line modal in production. So my hope in making this would be to help accomplish that update. Maybe we update that ticket into taking this and making it a standalone component?